src/client/datascience/common.ts

```diff
@@ -150,37 +149,14 @@ export function generateNewNotebookUri(
     }
 }
 
-export async function getRealPath(
-    fs: IFileSystem,
-    execFactory: IPythonExecutionFactory,
-    pythonPath: string,
-    expectedPath: string
-): Promise<string | undefined> {
-    if (await fs.localDirectoryExists(expectedPath)) {
+export async function tryGetRealPath(expectedPath: string): Promise<string | undefined> {
```

Comment:
I put this back to wrap the nodejs realpath func. It was throwing during the functional tests just searching for jupyter paths.